blueocean-dashboard/src/main/js/creation/github/GithubCreationApi.js

```diff
@@ -0,0 +1,77 @@
+import { Fetch } from '@jenkins-cd/blueocean-core-js';
+
+function delayedPromise(value, delay = 2000) {
+    return new Promise(resolve => {
+        setTimeout(() => {
+            resolve(value);
+        }, delay);
+    });
+}
+
+export default class GithubCreationApi {
+
+    constructor(fetch) {
+        this._fetch = fetch || Fetch.fetchJSON;
+    }
+
+    listOrganizations() {
+        return delayedPromise([
+            { name: 'cliffmeyers' },
+            { name: 'cloudbees' },
+            { name: 'jenkinsci' },
+        ]);
+    }
+
+    // eslint-disable-next-line
+    listRepositories(organization) {
+        return delayedPromise([
+            { name: 'blueocean-plugin' },
+            { name: 'jenkins-design-language' },
+            { name: 'blueocean-acceptance-test' },
+        ]);
+    }
+
+    createPipeline(organization, repository) {
+        return this._createItem(organization, repository);
+    }
+
+    createOrgFolder(organization) {
+        return this._createItem(organization);
+    }
+
+    // eslint-disable-next-line
+    _createItem(org, repo) {
+        return delayedPromise({}, 10000);
+
+        /*
```

Comment:
Yep, all Github code is test stuff, but I really needed to be able to test two two providers at once (probably should have left it out of this branch).  I disabled Github in this branch to avoid confusion: 4bb76f0